en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
+    3. Click **Save**.
+
+8. Add a ForEach statement as follows to filter the dates where the active cases have exceeded 5,000:
 
     1. Click the last **+** icon in your low-code diagram.
     
     2. Click **ForEach**.
 
     3. In the **Iterable Expression** field, enter `jsonArray`. Then click **Save**.
     
-    4. To apply the filter mentioned, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
+    4. To apply the filter, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
 
         ![Add Custom Statement After Foreach Statement](../assets/img/services/add-custom-statement-after-foreach-statement.png)
     
-    5. In the form that appears, click **Other**. Then enter the following in the **Statement** field.
+    5. In the form that appears, click **Other**. Then enter the following in the **Statement** field:
 
         ```ballerina
         int active = <int>(check item.Active);
         if (active > 5000) {
             response.push(item);
         }
         ```
-        
-        Then click **Save**.
-    
-9. To add a statement of the `Respond` type so that the result of the above processing is returned as a response, click the last **+** icon in your low-code diagram.
+    6. Click **Save**.
 
-    ![Add Respond Statement](../assets/img/services/add-respond-statement.png)
-    
-    In the **Respond Expression** field, enter `response`, and then click **Save**.
+9. Add a Respond statement as follows so that the result of the above processing is returned as a response:
+
+    1. Click the last **+** icon in your low-code diagram.
+
+        ![Add Respond Statement](../assets/img/services/add-respond-statement.png)
+
+    2. In the **Respond Expression** field, enter a response, and then click **Save**.
 
-    
 Now you have completed designing your service. It looks as follows.
 
-![No Code View](../assets/img/services/choreo-service-low-code-view.png)
+- In the low-code view
 
-The code view looks as follows.
+   ![No Code View](../assets/img/services/choreo-service-low-code-view.png)
 
-![Code View](../assets/img/services/choreo-service-code-view.png)
+- In the code view
 
+   ![Code View](../assets/img/services/choreo-service-code-view.png)
     
 To validate the service, click **Run & Test**. The following is logged to indicate that you have successfully started the service.
 
-![Service Started Log](../assets/img/services/service-started-notification.png)
+ ![Service Started Log](../assets/img/services/service-started-notification.png)
     
 Congratulations! You have successfully created your first service.
    
 ## Step 2: Test the service
 
-To test the `covid-stats` service you created, follow the procedure below:
+To test the `covid-stats` service you created, follow this procedure:
```

Comment:
shouldn't this be bold?

Reply:
We are using bold for field names and other occurrences of UI strings.